apps/library/lib/library/game_sup.ex

```diff
@@ -0,0 +1,46 @@
+defmodule Library.GameSup do
+  @moduledoc """
+  A supervisor for starting games
+  """
+
+  use Supervisor
+  alias Library.Game
+
+  def add_game(game_name) do
+    Supervisor.start_child(__MODULE__, [game_name])
+  end
+
+  # [g] Might use this approach to send a message (with all the games)
+  #     back to the phoenix channel that is creating the game.
+  def add_game(pid, game_name) when is_pid(pid) do
+    with {:ok, pid} <- Supervisor.start_child(__MODULE__, [game_name]),
+         do: send(pid, {:all_games, games})
+  end
+
+  def delete_game(game_name) do
+    Supervisor.terminate_child(__MODULE__, game_name)
+  end
+
+  def find_game(game_name) do
+    Enum.find games, fn(child) ->
+      Game.name(child) == game_name
```

Comment:
I wonder whether we should have an `in-progress` flag on games once they've started, and only allow joining those that haven't started?